src/server/worker/master.go

```diff
@@ -244,7 +244,7 @@ func (a *APIServer) serviceSpawner(pachClient *client.APIClient) error {
 		data := df.Datum(0)
 		logger, err := a.getTaggedLogger(pachClient, job.ID, data, false)
 		puller := filesync.NewPuller()
-		dir, err := a.downloadData(pachClient, logger, data, puller, nil, &pps.ProcessStats{}, nil, "")
+		dir, err := a.downloadData(pachClient, logger, data, puller, nil, &pps.ProcessStats{}, nil, "", a.isEnterpriseEnabled(logger))
```

Comment:
I think it'd probably be reasonable to store whether or not enterprise is enabled (the variable name should be `exportStats` or something, as discussed before) as a field on APIServer that gets set once on startup. If people enable enterprise they can just scale up and down their pipelines or delete pods or something. And that transition should be quite rare so I don't think it's worth it to do the check for each datum.

Reply:
Yea, I made the change, and it made the diff smaller :+1: